core-services/prow/02_config/_config.yaml

```diff
@@ -638,6 +638,7 @@ tide:
     - openshift/ovirt-csi-driver
     - openshift/ovirt-csi-driver-operator
     - openshift/ovn-kubernetes
+    - openshift/procfs
```

Comment:
This will enable what you want only for {openshift/release}-4.X branches and not for `master/main`. You will need to add the repo in that query as well.

Reply:
Done, ty